llvm/include/llvm/IR/Type.h

```diff
@@ -165,8 +165,8 @@ class Type {
   bool isPPC_FP128Ty() const { return getTypeID() == PPC_FP128TyID; }
 
   /// Return true if this is a well-behaved IEEE-like type, which has a IEEE
-  /// compatible layout as defined by APFloat::isIEEE(), and does not have
-  /// non-IEEE values, such as x86_fp80's unnormal values.
+  /// compatible layout, and does not have non-IEEE values, such as x86_fp80's
```

Comment:
"IEEE compatible layout" should probably be defined here along the lines of "sign, exponent, significand bits are in the expected locations."

Reply:
See https://github.com/llvm/llvm-project/pull/130557. Now we check this property explicitly.

Reply:
The documentation still needs a definition of "IEEE compatible layout", given that "as defined by APFloat::isIEEE" (which was providing it) is being deleted in this PR.

Reply:
Fixed in https://github.com/llvm/llvm-project/pull/130557/commits/dde7d59cf32c2fe0c3575639d59f35515e4c1e0a.